src/HealthChecks.Aws.Sns/DependencyInjection/SnsHealthCheckBuilderExtensions.cs

```diff
@@ -0,0 +1,45 @@
+using HealthChecks.Aws.Sns;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Extensions.DependencyInjection
```

Comment:
Use file scoped namespaces for all new files please.

Reply:
Fixed